src/index.ts

```diff
@@ -1,795 +1,15 @@
-    attestationKey: "Option<KeyIdOf>",
-    // new
-    publicKeys: "DidPublicKeyMap",
-    serviceEndpoints: "Option<ServiceEndpoints>",
-    lastTxCounter: "u64",
-  },
-  ServiceEndpoints: {
-    contentHash: "Hash",
-    // new
-    urls: "BoundedVec<Url, MaxEndpointUrlsCount>",
-    contentType: "ContentType",
-  },
-  MaxUrlLength: "u32",
-  MaxEndpointUrlsCount: "u32",
-  StorageError: {
-    _enum: {
-      DidAlreadyPresent: "Null",
-      DidNotPresent: "Null",
-      DidKeyNotPresent: "DidVerificationKeyRelationship",
-      VerificationKeyNotPresent: "Null",
-      CurrentlyActiveKey: "Null",
-      MaxTxCounterValue: "Null",
-      // new
-      MaxPublicKeysPerDidKeyIdentifierExceeded: "Null",
-      MaxTotalKeyAgreementKeysExceeded: "Null",
-      MaxOldAttestationKeysExceeded: "Null",
-    },
-  },
-};
-
-export const types21: RegistryTypes = {
-  ...types20,
-
-  StorageError: {
-    _enum: {
-      DidAlreadyPresent: "Null",
-      DidNotPresent: "Null",
-      DidKeyNotPresent: "DidVerificationKeyRelationship",
-      VerificationKeyNotPresent: "Null",
-      CurrentlyActiveKey: "Null",
-      MaxTxCounterValue: "Null",
-      MaxPublicKeysPerDidKeyIdentifierExceeded: "Null",
-      // renamed
-      MaxTotalKeyAgreementKeysExceeded: "Null",
-      MaxOldAttestationKeysExceeded: "Null",
-    },
-  },
-  // removed
-  DidUpdateDetails: undefined,
-  DidCreationDetails: {
-    did: "DidIdentifierOf",
-    newKeyAgreementKeys: "DidNewKeyAgreementKeys",
-    newAssertionMethodKey: "Option<DidVerificationKey>",
-    newCapabilityDelegationKey: "Option<DidVerificationKey>",
-    newServiceEndpoints: "Option<ServiceEndpoints>",
-  },
-  DidDetails: {
-    authenticationKey: "KeyIdOf",
-    keyAgreementKeys: "DidKeyAgreementKeys",
-    // renamed
-    capabilityDelegationKey: "Option<KeyIdOf>",
-    // renamed
-    assertionMethodKey: "Option<KeyIdOf>",
-    publicKeys: "DidPublicKeyMap",
-    serviceEndpoints: "Option<ServiceEndpoints>",
-    lastTxCounter: "u64",
-  },
-  DelegateSignatureTypeOf: "DidSignature",
-  ContentType: {
-    _enum: ["application/json", "application/ld+json"],
-  },
-
-  // fix: generics mostly don't work here, but OrderedSet is reduced to a Vec anyway
-  OrderedSet: undefined,
-  Collator: {
-    id: "AccountId",
-    stake: "Balance",
-    // fix
-    delegators: "Vec<Stake>",
-    total: "Balance",
-    state: "CollatorStatus",
-  },
-  Delegator: {
-    // fix
-    delegations: "Vec<Stake>",
-    total: "Balance",
-  },
-};
+import type { OverrideBundleDefinition } from "@polkadot/types/types";
+
+import { types8 } from "./types_8";
+import { types9 } from "./types_9";
+import { types10 } from "./types_10";
+import { types12 } from "./types_12";
+import { types17 } from "./types_17";
+import { types18 } from "./types_18";
+import { types19 } from "./types_19";
+import { types20 } from "./types_20";
+import { types21 } from "./types_21";
+import { types23 } from "./types_23";
```

Comment:
these still need to be exported from here, so a third-party can use them individually (what we do in most applications)

Reply:
fixed? 09fe5c3

Reply:
almost